rapport.qmd

```diff
@@ -142,11 +301,24 @@ g, H = get_gH(name, 1.0e-3)
 ## Validation de la méthode de la plus forte pente
 
 ```{julia}
-# votre code ici
+x_star11 = steepest_qp(g15, H15)
+x_star12 = steepest_qp(g75, H75)
+x_star13 = steepest_qp(g100, H100)
```

Comment:
Ton rapport fait 40 pages ... ce serait une bonne idée de n'afficher que les 100 itérations, p.ex.